com.yktoo.ymuse.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.yktoo.ymuse
+default-branch: master
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: ymuse
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+rename-icon: ymuse
+rename-desktop-file: ymuse.desktop
+modules:
+  - name: ymuse
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      build-args:
+        - --share=network
+    build-commands:
+      - go mod download
```

Comment:
there is no network access. you need to generate a manifest for the dependencies. See go-get in https://github.com/flatpak/flatpak-builder-tools

Reply:
Sorry this one I failed to resolve, see flatpak/flatpak-builder-tools#285. I'm not sure how to proceed, I'm unable to prepare a working sets of sources as Go still tries to download the modules.

Reply:
I wish I knew how to improve that... This is how I achieved it (crazy, I know):

```bash
 go mod download -json |
     grep '"Zip"' |
     cut -d '"' -f 4 |
     sed -E 's|.*?download/|https://proxy.golang.org/|' |
     while read u; do
         echo -e "- type: archive\n  url: $u\n  strip-components: 0\n  dest: vendor\n  sha256: $(curl -fsS "$u" | sha256sum -b | cut -d ' ' -f 1)\n"
     done
```